app/server/views/index.py

```diff
@@ -17,59 +17,6 @@ def get_js_bundle_filename():
 
     return Path(latest_file).name
 
-@index_view.route('/')
-def index():
-
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-
-@index_view.route('/transfers')
-def transfers():
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-
-@index_view.route('/accounts')
-def accounts():
-    return render_template('index.html', js_bundle_main=get_js_bundle_filename())
-
-@index_view.route('/accounts/<account_id>')
-def single_account(account_id):
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-@index_view.route('/users/<user_id>')
-def single_user(user_id):
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-@index_view.route('/users/<user_id>/verification')
-def single_user_verification(user_id):
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-@index_view.route('/upload')
-def upload():
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-# @index_view.route('/<account_type>/upload/')
-# def upload(account_type):
-#     return render_template('index.html')
-
-
-@index_view.route('/deprecatedVendor')
-def deprecatedVendor():
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-
-@index_view.route('/settings')
-def settings():
-    return render_template('index.html', js_bundle_main=get_js_bundle_filename())
-
-@index_view.route('/settings/<subroute>')
-def settings_subroute(subroute):
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
-@index_view.route('/activate-account/')
-def activate_account():
-    return render_template('index.html', js_bundle_main = get_js_bundle_filename())
-
```

Comment:
Are we sure this works @EDsCODE @tristanhcole? I recall we had only condensed routing back in the day, but we added the above stuff either because of:
a) We did some light server side rendering for some of the pages for SEO reasons (which we don't currently and so if that's the only reason this is here, then it can definitely be removed)
b) React was doing something funky and couldn't find the pages through a catch all during certain actions (like navigating directly to a url, rather than via a link from the index page)

I'm pretty sure it was (a), but just wanna be sure.

Reply:
https://github.com/teamsempo/SempoBlockchain/pull/189

Yes, there aren’t any issues with it. We also don’t do any server side rendering anymore so no reason to leave in. See original issue here https://github.com/teamsempo/SempoBlockchain/pull/182#discussion_r390772223